pages/common/complete.md

```diff
@@ -0,0 +1,15 @@
+# complete
+
+> Provides argument autocompletion to shell commands.
+
+- Apply an autocomplete function to a command:
+
+`complete -F {{function}} {{command}}`
+
+- Apply an autocomplete command to a command:
```

Comment:
Admittedly this phrasing isn't great, so I would love to hear alternatives.

Reply:
"autocomplete sub-command" ?

Reply:
@sbrl , @pxgamer - comments ?

Reply:
I was just trying to actually work out what this does as I had no idea. Done a bit of testing and I understand now, but I have no idea for how it should be worded.

Reply:
I've been thinking about this a bit.. Some thoughts:

The core issue here is the fact that two identical nouns located together may create confusion. That being said, either command can hypothetically be an absolute path, on the `$PATH` variable, or any other way we access commands in the shell. Since they are indeed the same thing, albeit one with autocompletion functionality, it may not be erroneous to use the same word. So instead, perhaps we can add an adjective to the latter command or update the first adjective. Perhaps something like this:

```markdown
- Apply a command that performs autocompletion to another command:
```

```markdown
- Apply an autocomplete command to a different command
```

Thoughts?

Reply:
What about `Apply an autocomplete provider (function/command) to a specified command name`? (replace in brackets as appropriate)

Reply:
This is a step in the right direction, but I'm still worried `provider` obfuscates the meaning. In addition, having `function/command` might be confusing considering the two options, `-F` and `-C`, specifically delineate between function and command. :grimacing: I promise I'm not trying to be difficult! Hehe. Just hoping we can find the clearest explanation here.

Reply:
> - Apply a command that performs autocompletion to another command:

I like this. Although slightly longer, it is perfectly clear what's happening.